trackers/huggingface/graphs.py

```diff
@@ -613,3 +626,165 @@ def results_table(df: pd.DataFrame):
+
+
+def device_funnel(df: pd.DataFrame) -> None:
+    """
+    Show count of how many models compile, assemble, etc
+    """
+    summ = DeviceStageCount(df)
+
+    stages = [
+        "All models",
+        "Convert to ONNX",
+        "Optimize ONNX file",
+        "Converts to FP16",
+        "Builds",
+    ]
+    cols = st.columns(len(stages))
+
+    for idx, stage in enumerate(stages):
+        with cols[idx]:
+            st.markdown(stage)
+
+    # Show Sankey graph with percentages
+    sk_val = {
+        "All models": f"{summ.all_models} models - 100%",
+        "Convert to ONNX": f"{summ.base_onnx} models - "
+        + str(int(100 * summ.base_onnx / summ.all_models))
+        + "%",
+        "Optimize ONNX file": f"{summ.optimized_onnx} models - "
+        + str(int(100 * summ.optimized_onnx / summ.all_models))
+        + "%",
+        "Converts to FP16": f"{summ.fp16_onnx} models - "
+        + str(int(100 * summ.fp16_onnx / summ.all_models))
+        + "%",
+        "Builds Nvidia": f"{summ.nvidia} models - "
+        + str(int(100 * summ.nvidia / summ.all_models))
+        + "% (Nvidia)",
+        "Builds Groq": f"{summ.groq} models - "
+        + str(int(100 * summ.groq / summ.all_models))
+        + "% (Groq)",
+        "Builds x86": f"{summ.x86} models - "
+        + str(int(100 * summ.x86 / summ.all_models))
+        + "% (x86)",
+    }
+    option = {
+        "series": {
+            "type": "sankey",
+            "animationDuration": 1,
+            "top": "0%",
+            "bottom": "20%",
+            "left": "0%",
+            "right": "19%",
+            "darkMode": "true",
+            "nodeWidth": 2,
+            "textStyle": {"fontSize": 16},
+            "nodeAlign": "left",
+            "lineStyle": {"curveness": 0},
+            "layoutIterations": 0,
+            "nodeGap": 12,
+            "layout": "none",
+            "emphasis": {"focus": "adjacency"},
+            "data": [
+                {
+                    "name": "All models",
+                    "value": sk_val["All models"],
+                    "itemStyle": {"color": "white", "borderColor": "white"},
+                },
+                {
+                    "name": "Convert to ONNX",
+                    "value": sk_val["Convert to ONNX"],
+                    "itemStyle": {"color": "white", "borderColor": "white"},
+                },
+                {
+                    "name": "Optimize ONNX file",
+                    "value": sk_val["Optimize ONNX file"],
+                    "itemStyle": {"color": "white", "borderColor": "white"},
+                },
+                {
+                    "name": "Converts to FP16",
+                    "value": sk_val["Converts to FP16"],
+                    "itemStyle": {"color": "white", "borderColor": "white"},
+                },
+                {
+                    "name": "Builds Nvidia",
+                    "value": sk_val["Builds Nvidia"],
```

Comment:
```suggestion
                    "name": "Executes Nvidia",
                    "value": sk_val["Executes Nvidia"],
```

?

Reply:
Changed `Builds Nvidia` to `Acquired Nvidia Perf` to match the suggestion on https://github.com/groq/mlagility/pull/187#discussion_r1129814417